### PR TITLE
Dlp

### PR DESCRIPTION
**Pull request by bethanyios**
Still need to do reflections

### CODE REVIEW

File: case-studies/DLP.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* Solutions should fit to what users use already - if users don't use apps, don't build an app; if they use Facebook, use Facebook.
+* Understanding familiarity with tech early allows us to tailor our programme for all participants. We've found that showing participants how to use Miro and other tools an effective way to keep everyone on board. Providing a quick demonstration of features can save a lot of time in explaining how to use them. 
+* Programme Design should always involve facilitators. In order to deliver an engaging programme in an informed way, facilitators ought to shape the programme. 
+* What makes sense to us, doesn't always to participants. We encourage usability testing of all prototypes and products in order to understand how people unfamiliar interact with what we've created. In the same way, we must consider the understanding of exercises we're familiar with as not obviously intuitive to participants. Explaning things in consise terms without jargon and with patience is key to success.
```
Comment (jc2820) [inline case-studies/DLP.md:59]:
```suggestion
* What makes sense to us, doesn't always to participants. We encourage usability testing of all prototypes and products in order to understand how people unfamiliar interact with what we've created. In the same way, we must consider the understanding of exercises we're familiar with as not obviously intuitive to participants. Explaining things in concise terms without jargon and with patience is key to success.
```

File: case-studies/DLP.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+n.b. Each workshop here also had time for sharing between the cohort of 9 charity organisations, or preparing for presentations / testing scripts
+
+---
+
+### Reflections
+
+* Having nine charities in the room provides fantastic opportunities for collaboration. Creating the space for communication across charities provides opportunity to reassure and provide solidarity within the cohort. 
+* Having a sounding board can be useful. Previously, we've found that not having a consistent lead on a project leads to lack of consistency and less value for charities participating in our programmes. However, on DLP, we found that charities who only had one individual attending could feel lost at times. Having a colleague to support would provide value opportunities to discuss ideas and increase confidence. We still believe that there should be at least one lead who attends all workshops, for consistency. See [this article](https://www.thecatalyst.org.uk/blog/the-key-role-of-the-charity-digital-lead) from The Catalyst to explore more. 
+* Solutions should fit to what users use already - if users don't use apps, don't build an app; if they use Facebook, use Facebook.
```
Comment (jc2820) [inline case-studies/DLP.md:56]:
```suggestion
* Solutions needn't always come in the form of a bespoke app. For example, if the current service uses Facebook, and users are shown to be happy with that format, find a solution that still uses Facebook, but improves the service in areas where it is lacking. 
```

**Review by jc2820 — CHANGES_REQUESTED** [review]
Just a couple of typos and a suggestion for one point that could perhaps be defined a little better.

File: case-studies/DLP.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+By the end of the first week, the participants already had a written script and conducted their user research. Early in the process, this put them in a position where they could make informed decisions about what part of the problem area was critical to their users. Keeping the user centricity at the heart of this programme, research and usability testing was continued throughout ensuring that the solutions were grounded in the needs of the users and what would really make impact.
+
+Moving forward from the programme, the organisations had three directions they could go in the next phase with CAST. Continuing with user research and discovery, using off-the-shelf solutions or a bespoke development - each organisation was guided towards being a position to follow one of these options based on what they had discovered about their problem area and user groups. They were also given the skillset and tools to be able to communicate about digital service design for other funding applications, having learnt the right language and understanding the process of development.
```
Comment (Albadylic) [inline case-studies/DLP.md:16]:
```suggestion
Moving forward from the programme, the organisations had three directions they could go in the next phase with Catalyst and The National Lottery Community Fund. Continuing with user research and discovery, using off-the-shelf solutions or a bespoke development - each organisation was guided towards being a position to follow one of these options based on what they had discovered about their problem area and user groups. They were also given the skillset and tools to be able to communicate about digital service design for other funding applications, having learnt the right language and understanding the process of development.
```

**Review by Albadylic — APPROVED** [review]
One small change, otherwise happy for this to merge - if you're happy with reflections, @bethanyios

File: case-studies/DLP.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* Understanding familiarity with tech early allows us to tailor our programme for all participants. We've found that showing participants how to use Miro and other tools an effective way to keep everyone on board. Providing a quick demonstration of features can save a lot of time in explaining how to use them. 
+* Programme Design should always involve facilitators. In order to deliver an engaging programme in an informed way, facilitators ought to shape the programme. 
+* What makes sense to us, doesn't always to participants. We encourage usability testing of all prototypes and products in order to understand how people unfamiliar interact with what we've created. In the same way, we must consider the understanding of exercises we're familiar with as not obviously intuitive to participants. Explaining things in concise terms without jargon and with patience is key to success.
+* Trust in the process is a key factor in reaching the goal. We've found that participants who second guess or hestitate to engage get less out of programmes than those who jump straight in with enthusiasm, even without knowing the destination.
```
Comment (Albadylic) [inline case-studies/DLP.md:60]:
```suggestion
* Trust in the process is a key factor in reaching the goal. We've found that participants who second guess or hesitate to engage get less out of programmes than those who jump straight in with enthusiasm, even without knowing the destination. 
```

**Review by Albadylic — APPROVED** [review]

**Pull request merged**